js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -82,6 +86,10 @@ function isMobile() {
     return devices.current().deviceType !== 'desktop';
 }
 
+function isCellInHeaderRow($cell) {
+    return !!$cell.closest(`.${HEADER_ROW_CLASS}`).length;
```

Comment:
Use parent(`.${HEADER_ROW_CLASS}`) instead of closest. It is much faster.

Reply:
ok